src/app/account/account.cpp

```diff
@@ -0,0 +1,63 @@
+#include "account.hpp"
+
+Account::Account(const std::string& user_name, const std::string& password)
+    : user_name_(user_name), password_hash_(hashSHA512(password)), map_of_random_questions_and_answers_{}
+{
+}
+
+bool Account::isPasswordProper(const std::string& login, const std::string& password)
+{
+    std::ifstream file("users/users.json");
+    nlohmann::json jsonFile;
+    file >> jsonFile;
+
+    std::string hashedPassword = hashSHA512(password);
+
+    for (auto& element : jsonFile)
+    {
+        if (element[login] == login && element[password] == hashedPassword)
+        {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+bool Account::areAnswersCorrect(const std::string& answer1, const std::string& answer2, const std::string& answer3)
+{
+    size_t judge{};
+    std::ifstream file("users/users.json");
+    nlohmann::json jsonFile;
+    file >> jsonFile;
+
+    for (const auto& user : jsonFile)
+    {
+        for (const auto& answer : user["security_questions"])
+        {
+            if (answer == answer1)
+            {
+                judge++;
+            }
+            if (answer == answer2)
+            {
+                judge++;
+            }
+            if (answer == answer3)
+            {
+                judge++;
```

Comment:
Please correct me if I understand that incorrectly.
Algorithm iterates over each user and then for each user security questions iterates and checks answers.
I find it inefficient as it will always iterate over whole container. 
Also how about having the same questions and answers for multiple users?
Then we might miss check. 

I suggest to first find expected user, then compare answers.

Answer comparison does a bit too much. 
Could it be simplified like this? 
`user["security_questions"][0] == answer0 &&
user["security_questions"][1] == answer1 &&
user["security_questions"][2] == answer2`